libs/backend/src/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -569,26 +578,19 @@ export function buildCastVoteRecord({
           buildOriginalSnapshot({
             castVoteRecordId,
             marks: [
-              ...pages[0].interpretation.markInfo.marks,
-              ...pages[1].interpretation.markInfo.marks,
+              ...interpretations[0].markInfo.marks,
+              ...interpretations[1].markInfo.marks,
             ],
             definiteMarkThreshold,
             election,
             electionOptionPositionMap,
           }),
         ],
-    BallotImage: hasImageFileUris
-      ? pages.map((page) =>
-          page.imageFileUri
-            ? {
-                '@type': 'CVR.ImageData',
-                Location: page.imageFileUri,
-              }
-            : {
-                // empty object to represent a page with no included image
-                '@type': 'CVR.ImageData',
-              }
-        )
+    BallotImage: imageFileUris
+      ? imageFileUris.map((imageFileUri) => ({
+          '@type': 'CVR.ImageData',
+          Location: imageFileUri,
+        }))
```

Comment:
Following up on our discussion in https://github.com/votingworks/vxsuite/pull/3988#discussion_r1333360383 re including either no images or both front and back images, but never just one image